pkg/storage/segments/repairer.go

```diff
@@ -67,7 +67,8 @@ func (repairer *Repairer) Repair(ctx context.Context, path storj.Path) (err erro
 	expiration := pointer.GetExpirationDate()
 
 	var excludeNodeIDs storj.NodeIDList
-	var healthyPieces []*pb.RemotePiece
+	var healthyPieces, unhealthyPieces []*pb.RemotePiece
+	healthyMap := make(map[int32]bool)
```

Comment:
I wonder if we could combine `healthyPieces` and `healthyMap` into a single `map[int32]*pb.RemotePiece`?